### PR TITLE
Part1a attempt

### PR DESCRIPTION
**Pull request by wadeevans**
Please review Part1a of Project3

### CODE REVIEW

**Review by matkatmusic — APPROVED** [review]

**Comment by wadeevans** [comment]
Please review Part 1b

File: main.cpp [hunk 1]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+5 properties:
+    1) Temperature of Fridge
+    2) Temperature of Freezer
+    3) Available Space in Fridge
+    4) Available Space in Freezer
+    5) 
+3 things it can do:
+    1) Change Fridge Temperature
+    2) Change Freezer Temperature
+    3) Check Temperatures
```
Comment (matkatmusic) [inline main.cpp]: You are missing a property.
Feel free to change `Thing 4)` to a different object if it'll be easier to come up with 5 properties.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (wadeevans) [reply main.cpp]: I have amended Thing 4)

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]
If you think you can come up with 5 properties and 3 actions for each of these, you're welcome to use them.
```
    1) Lights
    2) Cooker
    3) Fridge
    4) Sink
    5) Toaster
```

**Comment by wadeevans** [comment]
Added Things 5 - 9. Properties of Thing 10

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**